docs/css/extra.css

```diff
@@ -4,33 +4,39 @@ main {
     max-width: 1000px;
     background-color: #ffffff;
     margin-left: -20px;
-    padding-left: 20px;
-    padding-top: 50px;
-    padding-right: 20px;
+    /* padding-left: 20px; */
+    /* padding-top: 50px;
+    padding-right: 20px; */
     font-size: 18px
 }
 
 @media all 
-    and (min-width: 600px) 
-    and (max-width: 1200px) {
+    and (min-width: 770px) 
+    and (max-width: 1063px) {
```

Comment:
These seem very specific. Is there a reason?

Reply:
readthedocs has built in styling and autoscaling for different browser sizes. 768 is when readthedocs switches the navigation bar to the top, but the larger number, I can't remember the reasoning. Looking into it.